doc/src/manual/embedding.md

```diff
@@ -13,6 +13,7 @@ We start with a simple C program that initializes Julia and calls some Julia cod
 
 ```c
 #include <julia.h>
+JULIA_DEFINE_FAST_TLS() // only define this once, in an executable (not in a shared library) if you want fast code.
```

Comment:
Sure, it being linux-specific is only reflective of the current situation, will not necessarily always be the case. We should update documentation when the documented code changes in relevant ways, as usual.